pkg/config/config.go

```diff
@@ -487,8 +488,9 @@ type ImageConfig struct {
 	// containing credentials necessary for pulling images from secure
 	// registries.
 	GlobalAuthFile string `toml:"global_auth_file"`
-	// PauseImage is the name of an image which we use to instantiate infra
-	// containers.
+	// PauseImage is the name of an image on a registry which we use to instantiate infra
+	// containers. It should start with a registry host name.
+	// Most users should prefer ParsePauseImage().
```

Comment:
I think we can just mention that `PauseImage` gets validated on config validation.